Core/src/org/sleuthkit/autopsy/corecomponents/DataResultViewerTable.java

```diff
@@ -768,7 +767,7 @@ private void compactPropertiesMap() {
             return;
         }
 
-        // otherwise, move map the elemets into the available positions. 
+        // otherwise, move map elemets into the available positions. 
```

Comment:
"elements" is still misspelled

Reply:
@APriestman I guess I'm capable of multiple misspellings in one line of text! Fixed. I hope...